pkg/services/sqlstore/migrations/dashboard_mig.go

```diff
@@ -234,4 +234,13 @@ func addDashboardMigration(mg *Migrator) {
 	mg.AddMigration("Add isPublic for dashboard", NewAddColumnMigration(dashboardV2, &Column{
 		Name: "is_public", Type: DB_Bool, Nullable: false, Default: "0",
 	}))
+
+	mg.AddMigration("Add deleted for dashboard", NewAddColumnMigration(dashboardV2, &Column{
+		Name: "deleted", Type: DB_DateTime, Nullable: true,
+	}))
+
+	mg.AddMigration("Add index for deleted", NewAddIndexMigration(dashboardV2, &Index{
+		Cols: []string{"deleted"},
+		Type: IndexType,
+	}))
```

Comment:
I'm not sure that this index will ever actually be used, the database will always pick the best index based on the query conditions so it seems likely it'll prefer one of the indexes that start with org_id or folder_id

Reply:
I have checked the search query and it uses `Using intersect(IDX_dashboard_org_id,IDX_dashboard_deleted); Using where; Using filesort`

```
|  2 | DERIVED      | dashboard     | NULL       | index_merge | UQE_dashboard_org_id_uid,UQE_dashboard_org_id_folder_uid_title_is_folder,IDX_dashboard_org_id,IDX_dashboard_org_id_plugin_id,IDX_dashboard_is_folder,IDX_dashboard_deleted,IDX_dashboard_org_id_folder_id_title | IDX_dashboard_org_id,IDX_dashboard_deleted | 8,6     | NULL                                                  |  14583 |    11.11 | Using intersect(IDX_dashboard_org_id,IDX_dashboard_deleted); Using where; Using filesort |
```